src/Serilog.Sinks.RavenDB/Sinks/RavenDB/RavenDBSink.cs

```diff
@@ -47,12 +52,26 @@ public class RavenDBSink : PeriodicBatchingSink
         /// <param name="batchPostingLimit">The maximum number of events to post in a single batch.</param>
         /// <param name="period">The time to wait between checking for event batches.</param>
         /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
-        public RavenDBSink(IDocumentStore documentStore, int batchPostingLimit, TimeSpan period, IFormatProvider formatProvider)
+        /// <param name="defaultDatabase">Optional name of default database</param>
+        /// <param name="expirationTimeSpan">Optional time before a logged message will be expired assuming the expiration bundle is installed. Zero (00:00:00) means no expiration. If this is not provided but errorExpirationTimeSpan is, errorExpirationTimeSpan will be used for non-errors too.</param>
+        /// <param name="errorExpirationTimeSpan">Optional time before a logged error message will be expired assuming the expiration bundle is installed.  Zero (00:00:00) means no expiration. If this is not provided but expirationTimeSpan is, expirationTimeSpan will be used for errors too.</param>
+        public RavenDBSink(IDocumentStore documentStore, int batchPostingLimit, TimeSpan period, IFormatProvider formatProvider, string defaultDatabase = null, TimeSpan? expirationTimeSpan = null, TimeSpan? errorExpirationTimeSpan = null)
             : base(batchPostingLimit, period)
         {
             if (documentStore == null) throw new ArgumentNullException("documentStore");
             _formatProvider = formatProvider;
             _documentStore = documentStore;
+            _defaultDatabase = defaultDatabase;
+            _expirationTimeSpan = expirationTimeSpan;
+            _errorExpirationTimeSpan = errorExpirationTimeSpan;
+            if (_errorExpirationTimeSpan == null)
```

Comment:
Perhaps `_errorExpiration = _errorExpiration ?? _expiration` and the same below? Fairly common pattern to use null-coalescing with these.